backend/stories/crud.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy.orm import Session
+import json
+from . import models, schemas
+
+def get_story(db: Session, story_id: int):
+    db_story = db.query(models.Story).filter(models.Story.id == story_id).first()
+    return schemas.Story.from_module(db_story)
```

Comment:
✨  There's a small inconsistency I noticed here with the Users crud helpers while working on the auth - this method returns an instance of the pydantic model, not the SQLAlchemy model (which is what the examples showed, and what I'd done in `users.crud`). This meant it was particularly impractical for what I had to do because at one point - I really needed to get the DB model, so I had to basically duplicate this!

From what I gather of the use of pydantic and SQLAlchemy in FastAPI so far, and how things are distributed, I think the `crud` file is meant to be the interface with the database for any given app. Pydantic comes in in the inbound side of it (the parameters for the methods) as a way to validate we actually get dictionaries with the correct shape. However, I suspect we're not meant to use it internally once the data has been validated. The outbound side of things could be easily handled by the routers, for example (parsing the SQLAlchemy models to pydantic models).

I'm not saying we **have** to change it, but I'm putting it out there for discussion 😇 Sorry for the long rant, here's a potato 🥔

Reply:
I agree with your opinion. Also, when I did  this I wasn't sure pydantic would do the conversion by itself. So, I already removed the conversion :)